plugins/datadog-standalone/datadog-standalone.go

```diff
@@ -83,6 +83,8 @@ var (
 			}
 			if result.Status == "ok" {
 				log.Info("Successfully transmitted metrics")
+			} else if result.Status != nil {
```

Comment:
I was trying to avoid warning when status is absent, which from what I can make out of the [datadog api reference](http://docs.datadoghq.com/api/), it might be. So I feel like an warning will then end 'expected status to be ok, actually nil'. I guess that's not too awful though, what's your feeling?